service/raft/group0_state_machine.cc

```diff
@@ -57,6 +58,22 @@ static mutation convert_history_mutation(canonical_mutation m, const data_dictio
     return m.to_mutation(db.find_schema(db::system_keyspace::NAME, db::system_keyspace::GROUP0_HISTORY));
 }
 
+future<> write_mutations_to_database(storage_proxy& proxy, gms::inet_address from, std::vector<canonical_mutation> cms) {
+    std::vector<mutation> mutations;
+    mutations.reserve(cms.size());
+    try {
+        for (const auto& cm : cms) {
+            auto& tbl = proxy.local_db().find_column_family(cm.column_family_id());
+            mutations.emplace_back(cm.to_mutation(tbl.schema()));
+        }
+    } catch (replica::no_such_column_family& e) {
+        slogger.error("Error while applying topology mutations from {}: {}", from, e);
+        throw std::runtime_error(::format("Error while applying topology mutations: {}", e));
```

Comment:
The messages mention "topology mutations" but the function will be used for other mutations too

Reply:
Changed.